### ISSUE TITLE
Visual Studio Custom Action Build issue

### ISSUE THREAD

**Issue by Rakesh1234567** [wixtoolset/issues#6476]
## Bugs

If this issue is a bug:
I want to create and build Custom action with wix version 3.10. I have installed WixToolset 3.10.4 and created custom action after building it thows and error that required wix version 3.11 or newer.
[Defect.txt](https://github.com/wixtoolset/issues/files/6653049/Defect.txt)

* Which version of WiX are you building with?
> 3.10.4

* Which version of Visual Studio are you building with (if any)?

>2019

* Which version of the WiX Toolset Visual Studio Extension are you building with (if any)?

> Wix Toolset Visual Studio 2019 Extension

* Which version of .NET are you building with?

> 4.8

* If the problem occurs when installing your packages built with WiX, what is the version of Windows the package is running on?

> windows 10

* Describe the problem and the steps to reproduce it.

> Create new custom action library project and build the project

* Describe the behavior you expected and how it differed from the actual behavior.

> That project should build with wix version 3.10

**Comment by barnson** [comment]
Which template did you choose? What happens when you build it?

**Comment by Rakesh1234567** [comment]
Hi,
I have installed wix toolset 3.10.4 and vs extension to 2019. After these installations I have created Wix Custom Action Project and trying to build it then that throws an error "The WiX Toolset v3.11 (or newer) build tools must be installed to build this project. To download the WiX Toolset, see http://wixtoolset.org/releases/". But I want to use 3.10 only and create custom action.

Regards,
Rakesh
